cfpb/complaint-intake
# Made identify options a template

## Changes

- Made "how should this company identify you" section a template.
- Updated product option ("how should this company identify you") combinations.
- Updated company name selection to handle proper state when removing name.
- Adjusts margin of date of birth input to align it with other fields.

## Testing

- `gulp clean && gulp build`
- select a product and subproduct on step 1.
- on step 4 try any combination of: 
  - entering an existing company name.
  - entering a non-existing company name.
  - deleting the entered company name.
  - checking/unchecking "This company should also receive and respond to this complaint."
- Also on step 4, click "Yes" for "Additional company":
  - Select "Credit reporting"
  - Select either subproduct option.
  - Company name should not show up till subproduct is selected. 
  - Try same thing with "Money transfer …" and "Debt settlement"

## Review

- @niqjohnson 

## Screenshots

![screen shot 2016-07-14 at 10 21 21 am](https://cloud.githubusercontent.com/assets/704760/16842826/d923df04-49ac-11e6-8969-0bb9b5172b52.png)
